feature/isis/otg_tests/weighted_ecmp_test/weighted_ecmp_test.go

```diff
@@ -200,7 +214,30 @@ func TestWeightedECMPForISIS(t *testing.T) {
 	}
 	p3 := dut.Port(t, "port3")
 	gnmi.Await(t, dut, gnmi.OC().Interface(p3.Name()).OperStatus().State(), time.Minute*2, oc.Interface_OperStatus_DOWN)
+
+	if deviations.WecmpAutoUnsupported(dut) {
+		var weight string
+		switch dut.Vendor() {
+		case ondatra.CISCO:
+			weight = " router isis DEFAULT \n interface Bundle-Ether 2 \n address-family ipv4 unicast \n weight 200 \n address-family ipv6 unicast \n weight 200 \n ! \n interface Bundle-Ether 3 \n address-family ipv4 unicast \n weight 400 \n address-family ipv6 unicast \n weight 400 \n ! \n interface Bundle-Ether 4 \n address-family ipv4 unicast \n weight 400 \n address-family ipv6 unicast \n weight 400 \n"
```

Comment:
change in weights is not taking effect, traffic is still distributed as 33% each, please verify

Reply:
fixed on the latest PR